aws/resource_aws_spot_fleet_request_test.go

```diff
@@ -2264,27 +2264,16 @@ resource "aws_spot_fleet_request" "test" {
 
 func testAccAWSSpotFleetRequestLaunchSpecificationWithInstanceStoreAmi(rName string, rInt int, validUntil string) string {
 	return testAccAWSSpotFleetRequestConfigBase(rName, rInt) + fmt.Sprintf(`
-data "aws_ami" "ubuntu_instance_store" {
-  most_recent = true
-  owners      = ["099720109477"] # Canonical
-
-  # Latest Ubuntu 18.04 LTS amd64 instance-store HVM AMI
-  filter {
-    name   = "name"
-    values = ["ubuntu/images/hvm-instance/ubuntu-bionic-18.04-amd64-server*"]
-  }
-}
-
 resource "aws_spot_fleet_request" "test" {
     iam_fleet_role = "${aws_iam_role.test-role.arn}"
-    spot_price = "0.03"
+    spot_price = "0.05"
     target_capacity = 2
     valid_until = %[1]q
     terminate_instances_with_expiration = true
     wait_for_fulfillment = true
 
     launch_specification {
-      ami           = "${data.aws_ami.ubuntu_instance_store.id}"
+      ami           = data.aws_ami.amzn-ami-minimal-hvm-ebs.id
```

Comment:
This testing is specifically for instance store AMIs, so we will need to create a new AMI helper, e.g.

```go
// testAccLatestAmazonLinuxHvmInstanceStoreAmiConfig returns the configuration for a data source that
// describes the latest Amazon Linux AMI using HVM virtualization and an instance store root device.
// The data source is named 'amzn-ami-minimal-hvm-instance-store'.
func testAccLatestAmazonLinuxHvmInstanceStoreAmiConfig() string {
	return `
data "aws_ami" "amzn-ami-minimal-hvm-instance-store" {
  most_recent = true
  owners      = ["amazon"]

  filter {
    name   = "name"
    values = ["amzn-ami-minimal-hvm-*"]
  }

  filter {
    name   = "root-device-type"
    values = ["instance-store"]
  }
}
`
}
```

```suggestion
      ami           = data.aws_ami.amzn-ami-minimal-hvm-instance-store.id
```